Controller/Cart/Data.php

```diff
@@ -97,78 +69,14 @@ public function __construct(
-                throw new BoltException(__('Guest checkout is not allowed.'));
-            }
-
-            // flag to determinate the type of checkout / data sent to Bolt
-            $payment_only = $this->getRequest()->getParam('payment_only');
-            // additional data collected from the (one page checkout) page,
-            // i.e. billing address to be saved with the order
-            $place_order_payload = $this->getRequest()->getParam('place_order_payload');
-            // call the Bolt API
-            $boltpayOrder = $this->cartHelper->getBoltpayOrder($payment_only, $place_order_payload);
-
-            // format and send the response
-            $response = $boltpayOrder ? $boltpayOrder->getResponse() : null;
-
-            if ($response) {
-                $responseData = json_decode(json_encode($response), true);
-                $this->metricsClient->processMetric("order_token.success", 1, "order_token.latency", $startTime);
-            } else {
-                // Empty cart - order_token not fetched because doesn't exist. Not a failure.
-                $responseData['cart'] = [];
-            }
-
-            // get immutable quote id stored with cart data
-            list(, $cartReference) = $response ? explode(' / ', $responseData['cart']['display_id']) : [null, ''];
-
-            $cart = array_merge($responseData['cart'], [
-                'orderToken'    => $response ? $responseData['token'] : '',
-                'cartReference' => $cartReference,
-            ]);
-
-            if (isset($cart['currency']['currency']) && $cart['currency']['currency']) {
-                // cart data validation requirement
-                $cart['currency']['currency_code'] = $cart['currency']['currency'];
-            }
-
-            $hints = $this->cartHelper->getHints($cartReference, 'cart');
-
-            $result->setData([
-                'status' => 'success',
-                'cart' => $cart,
-                'hints' => $hints,
-                'backUrl' => '',
-            ]);
-        } catch (BoltException $e) {
-            $this->bugsnag->notifyException($e);
-            $result->setData([
-                'status' => 'success',
-                'restrict' => true,
-                'message' => $e->getMessage(),
-                'backUrl' => '',
-            ]);
-            $this->metricsClient->processMetric("order_token.failure", 1,"order_token.latency", $startTime);
-        } catch (Exception $e) {
-            $this->bugsnag->notifyException($e);
-
-            $result->setData([
-                'status' => 'failure',
-                'message' => $e->getMessage(),
-                'backUrl' => '',
-            ]);
-            $this->metricsClient->processMetric("order_token.failure", 1,"order_token.latency", $startTime);
-        } finally {
-            return $result;
-        }
+        $payment_only = $this->getRequest()->getParam('payment_only');
```

Comment:
Since we're refactoring anyways can we take the opportunity to replace snake_case with camelCase?

Reply:
Done